package.json

```diff
@@ -127,8 +127,8 @@
     "@storybook/react": "6.5.15",
     "@storybook/theming": "6.5.15",
     "@swc/cli": "0.1.62",
-    "@swc/core": "1.3.44",
-    "@swc/helpers": "0.4.14",
+    "@swc/core": "1.3.79",
+    "@swc/helpers": "0.5.1",
```

Comment:
Our packages currently depend on `@swc/helpers@0.4.x`, we shouldn't bump this until we bump our dependencies: https://github.com/microsoft/fluentui/blob/b130af3e12d5f46ff1217bcb47222eb89d55db2f/packages/react-components/react-tree/package.json#L54